### DISCUSSION TITLE
Login Failing with Valid Credentials

### DISCUSSION THREAD

**Discussion by RK721** [jhassan8/crunchyroll-tizen#91] [Q&A] [answered]
Trying both my email and username plus a correct password, I attempt to login, get a brief 1 second loading icon, then am returned to the same login credentials screen.  Was working 2 weeks ago, not sure why it isn't now.  Anyone else having this issue?

**Comment by RK721** [comment]
Update 1. Network logging preview produced:

{code: "auth.obtain_access_token.client_inactive", context: [], error: "invalid_client"}
  code: "auth.obtain_access_token.client_inactive"
  context: []
  error: "invalid_client"

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/28994398/eb4af82a-37cf-4096-8b50-0ad44133b064)
Let me know if any other info would be helpful.  From my limited understanding, I think there may be token issues at play here.

Update 2. Clearing app cache and data made produced no change

**Reply by jhassan8** [reply, marked as answer]
hi @RK721, what version of the app are you using? In version v1.1.3 a problem arose when logging in that was solved in the next version, try to use the latest one, which today is v1.1.5, to know which version you are using you can write in the same console as the screenshot

`changelog.data.version`

![image](https://github.com/jhassan8/crunchyroll-tizen/assets/55010545/d7c3f2f0-4fa4-4a35-860f-cbc67a822a02)

If you are still having a problem, check your password or write to me at jhassan8@outlook.com, and I will give you an account so you can check if it works.

**Reply by RK721** [reply]
Yup that did it.  I was out of date.  What's the best way to stay up to date with new version releases?
